src/index.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter as Router } from 'react-router-dom';
-// import { Provider } from 'react-redux';
-// import BookContainer from './components/BookContainer';
-// import store from './redux/configureStore';
```

Comment:
Please do not leave unused code as a comment in your codebase. Unused code as a comment make your code not clean.

Reply:
Fixed ✔